src/resources/js/admin/help-page.js

```diff
@@ -84,10 +84,30 @@ tribe.helpPage = tribe.helpPage || {};
 			ajaxurl,
 			request,
 			function ( results ) {
+
 				if ( results.success ) {
 					obj.$system_info_opt_in_msg.html( "<p class='optin-success'>" + results.data + "</p>" );
 				} else {
-					obj.$system_info_opt_in_msg.html( "<p class='optin-fail'>" + results.data.message + " Code:" + results.data.code + " Status:" + results.data.data.status + "</p>" ); // eslint-disable-line max-len
+					var html = "<p class='optin-fail'>Something has gone wrong!</p>";
+
+					if ( results.data ) {
+						if ( results.data.message ) {
+							html += ' ' + results.data.message;
```

Comment:
Is there any reason why we're not wrapping the return in `<p>` here as we do with the rest?

Reply:
nope - I missed that one (added the wrappers as an afterthought) thanks!